src/Illuminate/Console/Scheduling/Event.php

```diff
@@ -209,13 +213,23 @@ public function buildCommand()
     }
 
     /**
+     * Determine if the current enviroment is windows
+     *
+     * @return boolean
+     */
+    protected function isWindowsEnvironment()
```

Comment:
This function should be a global function in the support helpers.php file probably.